#### Introduction
Across America, inequalities in regards to healthcare are still prevelant, particularly when it comes to child birth. This analysis will attempt to derive statistically some of the trends in child birth based on data from 2016 to 2021 in the US. 

#### Set up the notebook environment by importing the required libraries

In [6]:
import pandas as pd
import numpy as np

#### Read data

In [3]:
data = pd.read_csv(r"C:\Users\lsmit\OneDrive\Desktop\UsBirthsProject_111\usBirths.csv.csv")

In [4]:
data.head()

State State Abbreviation  Year Gender  \
0  Alabama                 AL  2016      F   
1  Alabama                 AL  2016      F   
2  Alabama                 AL  2016      F   
3  Alabama                 AL  2016      F   
4  Alabama                 AL  2016      F   

                Education Level of Mother  Education Level Code  \
0                       8th grade or less                     1   
1  9th through 12th grade with no diploma                     2   
2   High school graduate or GED completed                     3   
3   Some college credit, but not a degree                     4   
4               Associate degree (AA, AS)                     5   

   Number of Births  Average Age of Mother (years)  Average Birth Weight (g)  
0              1052                           27.8                    3116.9  
1              3436                           24.1                    3040.0  
2              8777                           25.4                    3080.0  
3              6453                           26.7                    3121.9  
4              2227                           28.9                    3174.3

### Variable Description
-State  
-State Abbreviation  
-Year  
-Gender  
-Education Level of the Mother  
-Education Level Code (the lower number equals less education)  
-Number of Births  
-Average age of Mother  
-Average Birth Weight  

**Features**: The dataset has the following data types that we will be using

**Categorical/Nominal: Categories that have no inherent order or ranking.** -State, Gender

**Ordinal: Categories that have a meaningful order or ranking, but the intervals between ranks are not uniform or meaningful.** -Education Level

**Interval: Numeric values where the intervals between values are meaningful and equal, but there is no true zero** - Year

**Ratio: Numeric values with a meaningful zero point and equal intervals between values.** -AVG birth weight, AVG age of mother, number of births

**Useless Data:** - Education Level Code, state abbreviation





####  Fetch some info about data by .info()

In [5]:
print("Shape of data is :",data.shape)
print("*"*100)
data.info()

Shape of data is : (5496, 9)
****************************************************************************************************
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 5496 entries, 0 to 5495
Data columns (total 9 columns):
 #   Column                         Non-Null Count  Dtype  
---  ------                         --------------  -----  
 0   State                          5496 non-null   object 
 1   State Abbreviation             5496 non-null   object 
 2   Year                           5496 non-null   int64  
 3   Gender                         5496 non-null   object 
 4   Education Level of Mother      5496 non-null   object 
 5   Education Level Code           5496 non-null   int64  
 6   Number of Births               5496 non-null   int64  
 7   Average Age of Mother (years)  5496 non-null   float64
 8   Average Birth Weight (g)       5496 non-null   float64
dtypes: float64(2), int64(3), object(4)
memory usage: 386.6+ KB


##### Observations:

1. Our data has 5496 rows and 9 columns, meaning we have data on 5496 births. 
2. All columns show non-null values, meaning we are missing no values.
3. All columns are appropriately typed, based on this fact, data is ready for analysis. 
4. float64(2): AVG age of mother, AVG birth weight
5. int64(3): Num of births, Education level code(not used), Year
6. object(4): State, State Abbreviation(not used), Gender, Education level of mother



##### Questions to ask the Data:

1. What state has the lowest average birth weight?
2. What state has the youngest mothers?
3. What state has the least educated mothers?
4. What state had the highest / lowest number of births?